src/jit/codegenxarch.cpp

```diff
@@ -1813,162 +1813,20 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
                     assert(structDesc.eightByteCount == CLR_SYSTEMV_MAX_EIGHTBYTES_COUNT_TO_PASS_IN_REGISTERS);
 
                     regNumber retReg0 = REG_NA;
-                    emitAttr size0 = EA_UNKNOWN;
-                    unsigned offset0 = structDesc.eightByteOffsets[0];
+                    unsigned __int8 offset0 = 0;
```

Comment:
As we discussed offline, the return instruction is always the last one in the function. After the last instruction there are no any other instruction that could use a killed register. Added a TODO in lowerxarch to track the issue of unbalanced src/dest when the calle with 2 register struct return producec 2 destinations. The return in this case need to change to have 2 sources.